src/ScorecardController.js

```diff
@@ -0,0 +1,132 @@
+$( document ).ready( function() {
+  var game = new Game();
+  updateScorecard(game.frames, game.cumulatives)
+
+  $( "#submit" ).on('keyup', function() {
+      if (event.keyCode === 13) {
+          $("#submit").click();
+      }
+  });
+
+  $( "#submit" ).on('click', function() {
+    rollString = document.getElementById("roll_score").value;
+    if (rollString === 'X') {
+      rollString = '10'
+    }
+    rollNum = parseInt(rollString)
+    game.add_score(rollNum);
+    updateScorecard(game.frames, game.cumulatives)
+    document.getElementById("roll_score").value = ""
+  });
+
+  function updateScorecard(frames, cumulatives) {
+    var table = document.getElementById("myTable");
+    old_tbody = table.tBodies[0];
+
+    var new_tbody = document.createElement('tbody');
+    populate_with_new_rows_to_9(new_tbody, frames, cumulatives);
+    populate_with_new_rows_over_9(new_tbody, frames, cumulatives);
+
+    table.replaceChild(new_tbody, old_tbody)
+  }
+
+  function populate_with_new_rows_to_9(new_body, frames, cumulatives) {
+    counter = 0
+    frames.forEach( function(frame) {
+      if (counter < 9) {
+        var [row, framenum, roll1, roll2, roll3, frameTotal] = insert_row(new_body)
+
+        framenum.innerHTML = counter + 1
+        add_roll_1(roll1, frame.scores[0])
```

Comment:
Try and avoid using magic numbers. If you have methods with numbers in them there is almost certainly an opportunity to refactor - add_roll should probably just be a single method, and the index of the roll to add could be a parameter.